File: src/main/java/com/rustem/simplecrudapplication/model/Customer.java
package com.rustem.simplecrudapplication.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;
import java.math.BigInteger;

@Data
@EqualsAndHashCode(callSuper = false)
@Accessors(chain = true)
@Entity
@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
@Table(name = "customers")
public class Customer extends BaseEntity {

    @Column(name = "first_name")
    private String firstName;

    @Column(name = "last_name")
    private String lastName;

    @Column(name = "address")
    private String address;

    @Column(name = "budget")
    private BigInteger budget;
}

File: src/main/java/com/rustem/simplecrudapplication/model/Customer.kt
package com.rustem.simplecrudapplication.model

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import java.math.BigInteger
import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Table

@Entity
@JsonIgnoreProperties("hibernateLazyInitializer", "handler")
@Table(name = "customers")
class Customer : BaseEntity() {
    @Column(name = "first_name")
    val firstName: String? = null

    @Column(name = "last_name")
    val lastName: String? = null

    @Column(name = "address")
    val address: String? = null

    @Column(name = "budget")
    val budget: BigInteger? = null
}